theme_graphene/views/customizations.xml

```diff
@@ -246,4 +252,50 @@
     </xpath>
 </template>
 
+<!-- ======== NUMBERS ======== -->
+ <template id="s_numbers" inherit_id="website.s_numbers" name="Graphene s_numbers">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="oe_img_bg o_bg_img_center pb56 pt56 s_parallax_no_overflow_hidden parallax s_parallax_is_fixed" remove="pb24 pt24" separator=" "/>
+        <attribute name="style">background-image: url('/web/image/website.s_numbers_default_image'); background-position: 50% 0%;</attribute>
+    </xpath>
+    <xpath expr="//section/div[hasclass('container')]" position="before">
+        <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_numbers_default_image'); background-position: 50% 75%;"/>
```

Comment:
Hello @xlu-odoo , `s_numbers_default_image` doesn't exist and it's not used by the configurator. Could you use the `website.s_parallax_default_image` (or whatever) instead ?